cypress/integration/immutableDatabaseTests/services.ts

```diff
@@ -13,13 +13,7 @@
  *    See the License for the specific language governing permissions and
  *    limitations under the License.
  */
-import {
```

Comment:
Shouldn't this kind of thing be caught by the git hook prettier?

Reply:
Prettier currently only does the stuff in src, not cypress

Reply:
Can we expand it to cypress?

Reply:
Could, make an issue for it.  PR getting bloated already.

Reply:
Good idea, can do one bit formatting commit/PR that way, please create and link here for context

Reply:
dockstore/dockstore#3185